platform_secrets/config_factory.py

```diff
@@ -49,7 +48,9 @@ def _create_platform_auth(self) -> PlatformAuthConfig:
     def _create_kube(self) -> KubeConfig:
         endpoint_url = self._environ["NP_SECRETS_K8S_API_URL"]
         auth_type = KubeClientAuthType(
-            self._environ.get("NP_SECRETS_K8S_AUTH_TYPE", KubeConfig.auth_type.value)
+            self._environ.get(
+                "NP_SECRETS_K8S_AUTH_TYPE", KubeClientAuthType.CERTIFICATE
+            )
```

Comment:
```suggestion
            self._environ.get("NP_SECRETS_K8S_AUTH_TYPE", KubeConfig.auth_type.value)
```
why not to use KubeConfig.auth_type.value ?